environment.yml

```diff
@@ -44,3 +44,10 @@ dependencies:
         - Jinja2
         - stanza
         - shap==0.37.0
+        - jax
+        - jaxlib
+        - flax
+        - dalle-mini
+        - git+https://github.com/patil-suraj/vqgan-jax.git
```

Comment:
@jameswex do we have any bets practices about importing repos from GitHub like this?

Reply:
haven't had to do it before, but would be best to tag a specific commit/version to avoid breakage.